cnf-tests/pod-utils/hwlatdetect-runner/main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 	}
 
 	hwlatdetectArgs := []string{
-		hwlatdetectBinary,
+		"hwlatdetect",
```

Comment:
why this replace?

Reply:
We pass the binary path to the `exec` system call https://github.com/Tal-or/cnf-features-deploy/blob/cnftests_properly_isolate_cpus/cnf-tests/pod-utils/hwlatdetect-runner/main.go#L51 

It will detect automatically that it's a python program due to the shebang

Reply:
uhm, looking around it seems we're doing like others args are constructed - this instance used to be the odd one.
This comments applies to all the instances then. Why are we setting `argv[0] != argv0` all around?
Should not be a big deal 99% of the cases, but when causes bugs it will be quite hard to figure out.

Reply:
According to your comment `argv[0]` should contain the filename,  and this is exactly what we're doing here.
Unless filename implicitly means full path of filename

Reply:
yes, I meant the full absolute path. It's probably OK (or at very least not more broken then before) but AFAICT we're still drifting out of the expected behavior.